pages/about.tsx

```diff
@@ -196,14 +197,17 @@ const AboutPage = () => {
           Transaction calldata is immutable, and can be read with instructions{' '}
           <RelativeLink to="#35" title="CALLDATALOAD" />,{' '}
           <RelativeLink to="#36" title="CALLDATASIZE" />, and{' '}
-          <RelativeLink to="#37" title="CALLDATACOPY" />.
+          <RelativeLink to="#37" title="CALLDATACOPY" />. When a contract
+          executes an Xcall instruction, it also creates an internal
```

Comment:
rather than Xcall, wouldn't it be more clear to write xCALL?

Reply:
Good call :-)

Reply:
Done

Reply:
I don't see the change? You should change them all, not just one :-)